nautobot/ipam/models.py

```diff
@@ -936,7 +937,7 @@ class IPAddress(PrimaryModel, StatusModel, RoleModelMixin):
     objects = BaseManager.from_queryset(IPAddressQuerySet)()
 
     class Meta:
-        ordering = ("ip_version", "host", "prefix_length")  # address may be non-unique
+        ordering = ("ip_version", "host", "mask_length")  # address may be non-unique
```

Comment:
Should `ordering` also include something like `parent__namespace`? Or are we OK with IPAddresses in different Namespaces being interleaved?

Reply:
Yeah I think that makes sense. However, there is no way of seeing `parent__namespace` from `IPAddressListTable`. Would it make sense to add a column for that attribute?